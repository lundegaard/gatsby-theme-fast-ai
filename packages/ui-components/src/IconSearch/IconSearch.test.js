import React from 'react';
import { shallow } from 'enzyme';

import IconSearch from './IconSearch';

describe('<IconSearch />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<IconSearch {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
