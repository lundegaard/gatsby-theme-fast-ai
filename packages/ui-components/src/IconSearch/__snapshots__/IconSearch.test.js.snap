// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconSearch /> matches snapshot when shallowly rendered 1`] = `
<SvgIcon>
  <rect
    height="24"
    opacity="0"
    width="24"
  />
  <path
    d="M20.71 19.29l-3.4-3.39A7.92 7.92 0 0 0 19 11a8 8 0 1 0-8 8 7.92 7.92 0 0 0 4.9-1.69l3.39 3.4a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42zM5 11a6 6 0 1 1 6 6 6 6 0 0 1-6-6z"
  />
</SvgIcon>
`;
